code/src/components/album.css

```diff
@@ -0,0 +1,72 @@
+.album-cards {
+  margin: 10px;
+  width: 90%;
+  height: 30%;
+  position: relative;
+}
+
+.album-cover {
+  width: 100%;
+  max-height: 300px;
+}

```

Comment:
This needs some more work in mobile. There the images stretch which is not so nice. Try using `object-fit` here